src/controllers/user.controller.ts

```diff
@@ -9,4 +9,44 @@ export const createUser = async (req: Request, res: Response, next: NextFunction
     } catch (error) {
         next(error);
     }        
+};
+
+
+
+export const confirmUserEmail = async (req: Request, res: Response, next: NextFunction) => {
+    
+    const token = req.query.token;
+
+    if (!token) {
+        res.status(401).json({
+            message: "Unauthorized"
+        });
+        return;
+    }
+    
+    try {
+        await UserService.confirmUserEmail(String(token));
+        res.status(200).json({
+            message: "Email confirmed successfully."
+        });
+    } catch (error) {
+        next(error);
+    }
+    
+};
+
+
+export const userLogin = async (req: Request, res: Response, next: NextFunction) => {
+    
+    const {email, password} = req.body;
```

Comment:
@zaheerbabarkhan , since you are using Swagger to define and enforce a strict schema for your API, you should use Swagger to receive the data.

example:

`const requestBody = request.swagger.params.body.value;`